Local failure building tests -- test "web_client_HistomicsTK_annotations" does not exist

@jbeezley: @cdeepakroy and I get the error below when trying to build the tests on master. Clearly Travis has no problem with it, though. I get this error with a new build directory for Girder, and the particular revisions 6a7d593 and girder/girder@7b4022b. With `git bisect` I was able to locate bab64d8 as the first commit that causes the test build to fail.

```
-- Including plugin.cmake from "HistomicsTK"
CMake Error at plugins/HistomicsTK/plugin.cmake:100 (set_property):
  set_property given TEST names that do not exist:

    web_client_HistomicsTK_annotations

Call Stack (most recent call first):
  tests/CMakeLists.txt:111 (include)
  tests/CMakeLists.txt:123 (_add_plugin)


-- Configuring incomplete, errors occurred!
```

--- a/plugin.cmake
+++ b/plugin.cmake
@@ -14,6 +14,11 @@
 #  limitations under the License.
 ###############################################################################
 
+# support the "if(TEST ...)" command
+if(POLICY CMP0064)
+  cmake_policy(SET CMP0064 NEW)
+endif()
+
 function(add_histomicstk_python_test case)
   add_python_test("${case}" PLUGIN HistomicsTK
     COVERAGE_PATHS "${PROJECT_SOURCE_DIR}/plugins/HistomicsTK/histomicstk"
@@ -116,7 +121,7 @@ add_histomicstk_python_test(cli_results PLUGIN HistomicsTK
     # multiple tests, so add it in one of the tests for now
     # "plugins/HistomicsTK/Easy1.png"
 )
-if (TEST server_HistomicsTK.cli_results)
+if(TEST "server_HistomicsTK.cli_results")
   set_property(TEST server_HistomicsTK.cli_results APPEND PROPERTY ENVIRONMENT
       "CLI_LIST_ENTRYPOINT=${PROJECT_SOURCE_DIR}/plugins/slicer_cli_web/server/cli_list_entrypoint.py"
       "CLI_CWD=${PROJECT_SOURCE_DIR}/plugins/HistomicsTK/server"
@@ -131,7 +136,7 @@ add_web_client_test(
 )
 # Ideally, client tests would support the EXTERNAL_DATA keyword, but for now
 # we just use a data file used for one of the server tests.
-if (TEST web_client_HistomicsTK_annotations)
+if(TEST "web_client_HistomicsTK_annotations")
   set_property(
     TEST web_client_HistomicsTK_annotations APPEND PROPERTY
     ENVIRONMENT "GIRDER_TEST_DATA_PREFIX=${GIRDER_EXTERNAL_DATA_ROOT}"

--- a/plugin.cmake
+++ b/plugin.cmake
@@ -116,10 +116,12 @@ add_histomicstk_python_test(cli_results PLUGIN HistomicsTK
     # multiple tests, so add it in one of the tests for now
     # "plugins/HistomicsTK/Easy1.png"
 )
-set_property(TEST server_HistomicsTK.cli_results APPEND PROPERTY ENVIRONMENT
-    "CLI_LIST_ENTRYPOINT=${PROJECT_SOURCE_DIR}/plugins/slicer_cli_web/server/cli_list_entrypoint.py"
-    "CLI_CWD=${PROJECT_SOURCE_DIR}/plugins/HistomicsTK/server"
-)
+if (TEST server_HistomicsTK.cli_results)
+  set_property(TEST server_HistomicsTK.cli_results APPEND PROPERTY ENVIRONMENT
+      "CLI_LIST_ENTRYPOINT=${PROJECT_SOURCE_DIR}/plugins/slicer_cli_web/server/cli_list_entrypoint.py"
+      "CLI_CWD=${PROJECT_SOURCE_DIR}/plugins/HistomicsTK/server"
+  )
+endif()
 
 # front-end tests
 add_web_client_test(
@@ -129,10 +131,12 @@ add_web_client_test(
 )
 # Ideally, client tests would support the EXTERNAL_DATA keyword, but for now
 # we just use a data file used for one of the server tests.
-set_property(
-  TEST web_client_HistomicsTK_annotations APPEND PROPERTY
-  ENVIRONMENT "GIRDER_TEST_DATA_PREFIX=${GIRDER_EXTERNAL_DATA_ROOT}"
-)
+if (TEST web_client_HistomicsTK_annotations)
+  set_property(
+    TEST web_client_HistomicsTK_annotations APPEND PROPERTY
+    ENVIRONMENT "GIRDER_TEST_DATA_PREFIX=${GIRDER_EXTERNAL_DATA_ROOT}"
+  )
+endif()
 
 add_eslint_test(
   js_static_analysis_HistomicsTK "${PROJECT_SOURCE_DIR}/plugins/HistomicsTK/web_client"

